src/elements/TextInput.vue

```diff
@@ -61,20 +75,27 @@ const onChange = () => {
       <slot />
       <span v-if="required && model?.length === 0" class="required">*</span>
     </span>
-    <input
-      class="tbpro-input"
-      v-model="model"
-      :class="{'dirty': isDirty}"
-      :type="type"
-      :id="name"
-      :name="name"
-      :disabled="disabled"
-      :placeholder="placeholder"
-      :required="required"
-      @invalid="onInvalid"
-      @change="onChange"
-      ref="inputRef"
-    />
+    <span class="tbpro-input" :class="{ 'small-text': props.smallText }">
+      <span v-if="prefix" ref="inputPrefix" class="tbpro-input-prefix">{{
+        prefix
+      }}</span>
```

Comment:
My personal preference is, to keep the double curlies on the same line as the var name. This way, it's much easier for me to visually identify variables against plain text.

```suggestion
      <span v-if="prefix" ref="inputPrefix" class="tbpro-input-prefix">
        {{ prefix }}
      </span>
```